### ISSUE TITLE
Will be released a 5.3 version to include the new PSR middleware?

### ISSUE THREAD

**Issue by DominicDetta** [itsgoingd/clockwork#710]
Right now Im using inside the composer.json this kind dependency to download it: 

```json
"require": {
        "php": "^8.3",
        "itsgoingd/clockwork": "dev-master@dev",
```

**Comment by itsgoingd** [comment]
Hey, yes, it will be included in 5.3, but it might take some time until it's out.

**Comment by DominicDetta** [comment]
Thanks for clarification
